SUMMARY.md

```diff
@@ -70,6 +71,7 @@
   * [17.2 Logistic Model](ch17/classification_log_model.md)
   * [17.3 Logistic Cost](ch17/classification_cost.md)
   * [17.4 Using Logistic Regression](ch17/classification_log_reg.md)
+  * [17.5 Gradient Descent](ch17/classification_sgd.md)
```

Comment:
Can we rename this section to "Fitting a Logistic Model"? I think "Gradient Descent" alone is a bit too general (and repeats the title of Ch11).